.github/workflows/test.yaml

```diff
@@ -4,6 +4,24 @@ on:
   push:
 
 jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
We have `pnpm` in the README for site but also `yarn.lock` committed alongside `pnpm-lock.yaml`, so I think we should either use `pnpm` or `yarn` in CI and use the lock file

Reply:
Yes, that makes sense. I'll work on that! 👍 .

Reply:
Updating the linting dependency PR (#128) to use `yarn` instead of `pnpm`.